OmiseSDK/SourcePaymentMethod.swift

```diff
@@ -37,6 +37,51 @@ public enum PaymentInformation: Codable, Equatable {
 
     /// Online Alipay + Hongkong Wallet Payment Source
     case alipayHK
+    
+    /// The Atome customer information
+    public struct Atome: PaymentMethod {
+
+        public static var paymentMethodTypePrefix: String = OMSSourceTypeValue.atome.rawValue
+
+        public var type: String = OMSSourceTypeValue.atome.rawValue
+
+        /// The customers phone number. Contains only digits and has 10 or 11 characters
+        public let phoneNumber: String
+        public let name: String?
+        public let email: String?
+        public let shippingStreet: String?
+        public let shippingCity: String?
+        public let shippingCountryCode: String?
+        public let shippingPostalCode: String?
```

Comment:
street, country and postal code are not optional right?

Reply:
should we remove the question mark at the end? it means variable is nullable right?

Reply:
Yes, T? represents nullable value type. Fix this one as https://github.com/omise/omise-ios/pull/184/commits/9b8d40e9d71a50bb65c3ca74a6af9eb2817d2ac8. Thank you ka. :)